src/Tools/TransformXMLToJSONUsingXSLT/README.md

```diff
@@ -0,0 +1,72 @@
+# Transformation Tool from XML to Text
+
+Transformation Tool is an application to:.
+
+  - Transform files XML to Text
+  - Create text files
+  - Configure the entry files and set the out put files.
+
+### Technologies
+
+ - .Net FrameWork 4.7 
+ - XSLT Files to Transform.
+ - C# as language
+
+### Configuration
+
+Transformation Tool needs the next entries to work properly, this entries have to be into app.config file and they are in two sections which are mentioned below.
+
+**appSettings**
+This section is described in the table Files needed above.
+```sh
+<appSettings>
+    <add key="xmlFilePathResult" value="TestResult-Color_Tests.xml" />
+    <add key="xsltFile" value="TransformXMLToText.xslt" />
+    <add key="textFile" value="CustomTextResult.txt" />
+    <add key="textFileWithFilters" value="TextResultWithFilters.txt" />
+    <add key="applyFilters" value="1" /><!--1 means yes, 0 means not-->
+</appSettings>
+```
+**About TestResult-Color_Tests.xml**
+This file is in format NUnit2 and it is the result of a set of running test, there is an example above:
+Enter to the path where the console of NUnit where installed by default C:\Program Files (x86)\NUnit.org\nunit-console and the type the next command.
+```
+nunit3-console.exe "C:\\DSCoreNodesTests.dll" --where "test=="DSCoreNodesTests.ColorTests" and cat != Failure and cat != BackwardIncompatible" --labels=Before --explore="C:\\TestResult-Color_Tests.xml";format=nunit2
+```
+Where C:\\DSCoreNodesTests.dll is the Dll that contains the set of test, DSCoreNodesTests.ColorTests is the name space of the test and C:\\TestResult-Color_Tests.xml is the path and the name of the file where the result will be.
+
+
+
+### Files needed
+| File | Type | Description
+| ------ | ------ | ------ |
+| TransformXMLToText | xslt | File used to transform from XML to Text
+| CustomTextResult | txt | File where results of the convertion will be
+| textFileWithFilters | txt | File where results of the convertion will be but with filters 
+| TestResult-Color_Tests | xml | File´s name can be different because is the file to transform and it is the result of the tests 
+
+Note : The names of the files  can be changed if it is needed.
+
+**Filters**
+This section is the filters that will be applied in case the entry applyFilters has the value 1, in this case the slow tests are the ones that are filtered. 
```

Comment:
does this mean these tests will be filtered out or what? What is the point of applying a filter? Please add this to the readme.

Reply:
A diagram will be added to explain more about the tool and the context of the same.